src/blocks/Card.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import "./Card.scss"
+
+export interface CardHeaderProps {
+  /** Add divider between this and the next element */
+  divider?: "flush" | "inset"
+  /** An additional element(s) you can add to the side of the main child element(s) */
+  suffix?: React.ReactNode
+  /** Element ID */
+  id?: string
+  /** Additional class name */
+  className?: string
+  children: React.ReactNode
+}
+
+const CardHeader = (props: CardHeaderProps) => {
+  const classNames = ["card-header"]
+  if (props.className) classNames.push(props.className)
+
+  return (
+    <header id={props.id} className={classNames.join(" ")} data-divider={props.divider} data-flex-children={!!props.suffix}>
+      {props.children}
+      {props.suffix}
```

Comment:
This suffix feature is kind of weird to me. It could just be a personal preference, but I'd prefer all of the content to be in the children. In fact with your current setup when I do the following it looks the same as if I put it in the suffix.
```
      <Card.Header divider="flush" className="test-card-header">
        <>
          <h3>Wildflower</h3>
          <div>Second part of it</div>
        </>
      </Card.Header>
```
That's because when you have no suffix prop passed it outputs as `data-flex-children="false"` so the css is applied. 

I'll lean on the rest of the team on what syntax people prefer, but at minimum we should probably fix the issue where the css is only getting applied when we want it to.

Reply:
Yeah, I believe that was a holdover from a past attempt to work on the idea of adding an icon or close button or something else to the opposite end of the header, but we might just instead want a flex/grid child to get added rather than bake it in the Header. I'm fine with just taking that prop out.